corehq/apps/locations/models.py

```diff
@@ -114,6 +114,17 @@ class LocationType(models.Model):
     )  # levels below this location type that we start expanding from
     _expand_from_root = models.BooleanField(default=False, db_column='expand_from_root')
     expand_to = models.ForeignKey('self', null=True, related_name='+', on_delete=models.CASCADE)  # levels above this type that are synced
+    _include_without_expanding = models.ForeignKey(
+        'self',
+        null=True,
+        related_name='+',
+        db_column='include_without_expanding',
+        on_delete=models.CASCADE
+    )  # include all leves of this type and their ancestors
```

Comment:
MPTT stores a `level` property on all locations - would it be easier to store an integer here and sync all locations down to level `n`?